src/convnets/mobilenet.jl

```diff
@@ -208,71 +213,72 @@ Create a MobileNetv3 model.
-    (3, 3.67, 24, nothing,      relu, 1),
-    (5,    4, 40,       4, hardswish, 2),
-    (5,    6, 40,       4, hardswish, 1),
-    (5,    6, 40,       4, hardswish, 1),
-    (5,    3, 48,       4, hardswish, 1),
-    (5,    3, 48,       4, hardswish, 1),
-    (5,    6, 96,       4, hardswish, 2),
-    (5,    6, 96,       4, hardswish, 1),
-    (5,    6, 96,       4, hardswish, 1),
-  ], 
-  :large => [
-  #  k,   t,   c,      SE,         a, s
-    (3,   1,  16, nothing,      relu, 1),
-    (3,   4,  24, nothing,      relu, 2),
-    (3,   3,  24, nothing,      relu, 1),
-    (5,   3,  40,       4,      relu, 2),
-    (5,   3,  40,       4,      relu, 1),
-    (5,   3,  40,       4,      relu, 1),
-    (3,   6,  80, nothing, hardswish, 2),
-    (3, 2.5,  80, nothing, hardswish, 1),
-    (3, 2.3,  80, nothing, hardswish, 1),
-    (3, 2.3,  80, nothing, hardswish, 1),
-    (3,   6, 112,       4, hardswish, 1),
-    (3,   6, 112,       4, hardswish, 1),
-    (5,   6, 160,       4, hardswish, 2),
-    (5,   6, 160,       4, hardswish, 1),
-    (5,   6, 160,       4, hardswish, 1)
-  ]
-)
+mobilenetv3_configs = Dict(:small => [
+                               #  k,    t,  c,      SE,         a, s
+                               (3, 1, 16, 4, relu, 2),
+                               (3, 4.5, 24, nothing, relu, 2),
+                               (3, 3.67, 24, nothing, relu, 1),
+                               (5, 4, 40, 4, hardswish, 2),
+                               (5, 6, 40, 4, hardswish, 1),
+                               (5, 6, 40, 4, hardswish, 1),
+                               (5, 3, 48, 4, hardswish, 1),
+                               (5, 3, 48, 4, hardswish, 1),
+                               (5, 6, 96, 4, hardswish, 2),
+                               (5, 6, 96, 4, hardswish, 1),
+                               (5, 6, 96, 4, hardswish, 1),
+                           ],
+                           :large => [
+                               #  k,   t,   c,      SE,         a, s
+                               (3, 1, 16, nothing, relu, 1),
+                               (3, 4, 24, nothing, relu, 2),
+                               (3, 3, 24, nothing, relu, 1),
+                               (5, 3, 40, 4, relu, 2),
+                               (5, 3, 40, 4, relu, 1),
+                               (5, 3, 40, 4, relu, 1),
+                               (3, 6, 80, nothing, hardswish, 2),
+                               (3, 2.5, 80, nothing, hardswish, 1),
+                               (3, 2.3, 80, nothing, hardswish, 1),
+                               (3, 2.3, 80, nothing, hardswish, 1),
+                               (3, 6, 112, 4, hardswish, 1),
+                               (3, 6, 112, 4, hardswish, 1),
+                               (5, 6, 160, 4, hardswish, 2),
+                               (5, 6, 160, 4, hardswish, 1),
+                               (5, 6, 160, 4, hardswish, 1),
+                           ])
```

Comment:
Another minor point: this might be a case where we override the bot? Can we do that? Not an issue if not.

Reply:
So the comment can be fixed if we just keep comma separated characters. The formatter doesn't like it if the spaces aren't uniform and tries to fix it, but I think keep an ordered set of characters even if not lining up with the values below should be good enough>?

Reply:
I meant more that it adds an unnecessary amount of indentation.

Reply:
Oh. I searched but I don't find an alignment option _exactly_ for this just yet...I'll look some more

Reply:
No it's okay, this is very minor and I was just curious if we can override it. No reason to hold up the change.

Reply:
This alignment also seems contrary to the style guide. Perhaps it's not implemented yet in JuliaFormatter?